R/brdg_summary.R

```diff
@@ -432,6 +432,10 @@ pier_summary_continuous <- function(cards,
     dplyr::filter(.data$variable %in% .env$variables, !.data$context %in% "attributes") |>
     cards::apply_fmt_fn()
 
+  # if (all(cards_no_attr$context != "continuous")) {
+  #   stop("Something wrong")
+  # }
```

Comment:
here in the highlighted cases they are all categorical but we are in the continuous pier!!